src/App.tsx

```diff
@@ -15,36 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+
+function getPrepareGoods(
+  goods: string[],
+  { sortType, reverse }: { sortType: string; reverse: boolean },
+): string[] {
+  const preparedGoods: string[] = [...goods];
+
+  if (sortType === SORT_FIELD_ALPHABET) {
+    preparedGoods.sort((a, b) => a.localeCompare(b));
+  }
+
+  if (sortType === SORT_FIELD_LENGTH) {
+    preparedGoods.sort((a, b) => a.length - b.length);
+  }
+
+  if (reverse) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [sortType, setSortType] = useState<string>('');
```

Comment:
The initial state of `sortType` is set to an empty string. This could lead to confusion or errors if the empty state is not handled properly in the `getPrepareGoods` function. Consider setting a default sort type or explicitly handling the empty state in your sorting logic.